src/plugins/gr-he-common/vm/cloud_init.py

```diff
@@ -929,11 +933,34 @@ def _customization(self):
                     default=_('No')
                 ) == _('Yes').lower()
 
+            if (
+                self.environment[
+                    ohostedcons.CloudInit.APPLY_OPENSCAP_PROFILE
+                ] and self.environment[
```

Comment:
It might make sense to use queryEnvKey and slightly simplify this code, not sure